invenio_drafts_resources/services/records/service.py

```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
 #
-# Copyright (C) 2020 CERN.
-# Copyright (C) 2020 Northwestern University.
+# Copyright (C) 2021 CERN.
+# Copyright (C) 2021 Northwestern University.
```

Comment:
major: when updating copyright...
1- you should only modify that of the organization you work for (i.e. CERN in your case)
2- do NOT remove previous years, if existing make a range. Otherwise just add...

```suggestion
# Copyright (C) 2020-2021 CERN.
# Copyright (C) 2020 Northwestern University.
```